mep/people/templates/people/snippets/member_result.html

```diff
@@ -46,10 +46,9 @@
         <div class="member-dates">
             <dt>membership dates</dt>
             <dd>
-                {% if account_start %}{{ account_start }}{% endif %}
-                {% if account_end and account_start != account_end %}
-                 - <span class="sr-only">to</span>
-                {{ account_end }}{% endif %}
+            {% if account_start %}
+            {{ account_start }}{% endif %}{% if account_end and account_start != account_end %}–<span class="sr-only">to</span>{{ account_end }}
```

Comment:
Worth switching to the separator with aria label?
Do we have enough of these that it's worth a template snippet?

Reply:
it's a good question - i decided no because it's just so short, and also it feels a little weird to have stuff like the template for a particular widget depend on a snippet...idk why. but open to changing that.